File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        int k = 0;
       
        for (int i=0; i<nums.size(); i=i+1){
            if ((k < 2) || (nums[i] != nums[k-2])){
                nums[k] = nums[i];
                k+=1;
            }
        }
        return k;
        
    }
};

File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:


        k = 0
        for i in range(len(nums)):
            if k<2 or nums[i] != nums[k-2]:
                nums[k] = nums[i]
                k+=1
        return k
        